version/version.go

```diff
@@ -37,20 +47,44 @@ func Semver() string {
 	return version
 }
 
-// Println prints all available details about the current version in a
-// human-readable format, with an optional component name as the prefix
-func Println(component string) {
+func SemverWithEditionSuffix() string {
```

Comment:
What's the purpose of the edition suffix? Since we already output `community edition` or `enterprise edition`, I'm not totally sure of its purpose!

Reply:
This is part of the requirements from the trademark guidelines document. It must be included in the version - and to be clear what it stands for we also print out the full name.